sync/sync.go

```diff
@@ -211,3 +224,12 @@ func marathonTaskIdsSet(marathonApps []*apps.App) map[apps.TaskID]struct{} {
 	}
 	return tasksSet
 }
+
+func findTask(id apps.TaskID, tasks []*apps.Task) bool {
```

Comment:
This change will require to change `Marathoner.Tasks` answers from `[]*Task` to `[]Task`. I think it's good change since Task is small object (112 bytes) mostly pointers (4 strings and 2 arrays) so there should not be any problems. This change could be quite big and in multiple packages so I'd like to make it in separate PR #192